### PR TITLE
Add path assertions

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
This PR is a follow up to https://github.com/rubygems/rubygems/commit/f33345d6e62af339299d873d03cd3cfd14f9c989#diff-0c4b3e9996ebec38f191e68264665984R82.

### CODE REVIEW

**Comment by bronzdoc** [comment]
👀

File: lib/minitest/assertions.rb [hunk 1]
```diff
@@ -564,7 +572,7 @@ def refute test, msg = nil
     end
 
     ##
-    # Fails if +obj+ is empty.
+    # Fails if +obj+ is not empty.
```
Comment (zenspider) [inline lib/minitest/assertions.rb]: Not quite. This is either "Fails if obj is empty" or "Fails unless obj is not empty".

Think of it backwards: it's the opposite of `assert_empty` which "fails if obj is non-empty".

**Comment by zenspider** [comment]
Added and rearranged. Thank you!

File: lib/minitest/assertions.rb (hunk 1 above)
Comment (deivid-rodriguez) [reply lib/minitest/assertions.rb]: Right, I don't know what I was thinking :/. Thanks for getting this in.

**Pull request closed without merging**